.kokoro/environment/appengine_flex/common.cfg

```diff
@@ -0,0 +1,35 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Build logs will be here
+action {
+  define_artifacts {
+    regex: "**/*sponge_log.xml"
+  }
+}
+
+
+# Specify which tests to run
+env_vars: {
+    key: "ENVIRONMENT"
+    value: "appengine_flex"
+}
+
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-python"
```

Comment:
I'm currently authenticating against the shared python CI project. We should replace this before merging